backend/clickhouse/errors.go

```diff
@@ -698,6 +698,43 @@ func (client *Client) QueryErrorObjectsHistogram(ctx context.Context, projectId
 	return bucketTimes, totals, nil
 }
 
+func (client *Client) QueryErrorObjects(ctx context.Context, projectId int, errorGroupId int, count int, params modelInputs.QueryInput, page *int) ([]int64, int64, error) {
+	pageInt := 1
+	if page != nil {
+		pageInt = *page
+	}
+	offset := (pageInt - 1) * count
+
+	sb, err := readErrorsObjects(params, projectId)
+	if err != nil {
+		return nil, 0, err
+	}
+	sb.Where(sb.Equal("ErrorGroupID", errorGroupId))
+
+	sb.Select("ID, count() OVER() AS total")
+	sb.OrderBy("UpdatedAt DESC, ID DESC")
```

Comment:
should these be ordered by Timestamp? There could be a big delay between when an error instance happens and when it's finally saved in our db

Reply:
Good call! Updated here and the frontend to use the timestamp value as well